docs/algorithms/kem/kyber.yml

```diff
@@ -15,6 +15,8 @@ auxiliary-submitters:
 crypto-assumption: Module LWE+R with base ring Z[x]/(3329, x^256+1)
 website: https://pq-crystals.org/
 nist-round: 3
+standardization-status: Selected by NIST
```

Comment:
I'd disagree with this wording: AFAIK, Kyber (round 3 submission) is not identical with ML-KEM. So what about "Selected by NIST as the basis for standardization in ML-KEM (see below)"?